Extensions/TweenBehavior/tweenruntimebehavior.ts

```diff
@@ -107,6 +107,19 @@ namespace gdjs {
       return true;
     }
 
+    onDestroy() {
+      const shiftyJsScene = this._runtimeScene.shiftyJsScene;
+      if (!shiftyJsScene) return;
+
+      // Stop and delete all tweens of the behavior - otherwise they could:
+      // - continue to point to the behavior, and so to the object (memory leak),
+      // - affect the object in case it's recycled (wrong/hard to debug behavior).
+      for (const identifier in this._tweens) {
+        this._tweens[identifier].instance.stop();
```

Comment:
@jeremyckahn Quick question about Shifty (I did not have the time to check that in the source code before going to bed): is calling remove on the Shifty scene not enough to remove properly a tween? I had to call `stop` here (and also at the other place where a tween is removed), otherwise the tween would continue to run despite the call to remove. 

Is this normal/me not reading the docs or a potential bug? :)

Reply:
Hi @4ian, `Tweenable`s are **not** stopped when removed from a `Scene`. They are simply removed from the `Scene`'s data structure: https://github.com/jeremyckahn/shifty/blob/f634dd6f6aac6b25a2fea3d2b2b2184d9ba949ed/src/scene.js#L88-L96

Because the `remove` function returns the removed `Tweenable`, you should be able to do something like `scene.remove(tween).stop()`. Would that meet your needs?

As for whether this is the ideal behavior, it's a tough call. While I can see it being preferable to `stop` a removed tween most of the time, I can also imagine a scenario where that's not desired and the workaround would be a little awkward. The existing API makes it pretty easy to `stop` a removed tween, so it seems safer to leave it as-is.

Reply:
Sounds good to leave as is then, I was not sure if this was made on purpose or not! 👍